js/react_native/e2e/.detoxrc.js

```diff
@@ -38,7 +38,8 @@ module.exports = {
     simulator: {
       type: 'ios.simulator',
       device: {
-        type: 'iPhone 13',
+        type: 'iPhone 11',
```

Comment:
The highest iphone version that can support iOS 13 is iPhone 11.

Reply:
Still fails
```
error: Failed to find a device by type = "iPhone 11" and by OS = "iOS 13.0"
  
  HINT: Run 'applesimutils --list' to list your supported devices. It is advised only to specify a device type, e.g., "iPhone Xʀ" and avoid explicit search by OS version.
```